testimony.yaml

```
@@ -56,6 +56,7 @@ CaseComponent:
     - ForemanMaintain
     - Hammer
     - Hammer-Content
+    - HttpProxy
```

@vsedmik You might need to add this components.yml in CI repo to run this, and IIRC for CI we just remove spaces from component names provided in ohsnap for each team, so I'd suggest following a similar naming convention here, @devendra104 can also confirm this
```suggestion
    - HTTPProxy
```

---

I have the CI MR up already. I considered the `HttpProxy` better readable, but can change it if necessary. @devendra104 ?

---

I agree with you @vsedmik but good to keep it as similar to the Ohsnap's defined component name until something exceptional.

---

Updated, thanks for the feedback!